climada/util/coordinates.py

```diff
@@ -1198,7 +1351,12 @@ def _nearest_neighbor_approx(
     np.array
         with as many rows as coordinates containing the centroids indexes
     """
-
+    # first check that unit is in degree
+    if unit != "degree":
+        raise ValueError(
+            "Only degree unit is supported for nearest neighbor approximation."
+            "Please use euclidean distance for non-degree units."
```

Comment:
The docstring says `Compute the nearest centroid for each coordinate using the
    euclidean distance`. But the error says that one should change to Euclidian distance. I think there is a mix-up here.